plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/MigrateKVMAsync.java

```diff
@@ -51,19 +76,19 @@ public MigrateKVMAsync(final LibvirtComputingResource libvirtComputingResource,
 
     @Override
     public Domain call() throws LibvirtException {
-        long flags = 1 << 0;
+        long flags = VIR_MIGRATE_LIVE;
 
         // set compression flag for migration, if libvirt version supports it
         if (dconn.getLibVirVersion() >= 1000003) {
-            flags |= 1 << 11;
+            flags += VIR_MIGRATE_COMPRESSED;
         }
 
         if (migrateStorage) {
-            flags |= 1 << 6;
+            flags += VIR_MIGRATE_NON_SHARED_DISK;
```

Comment:
Really liked these changes :thumbsup:

Reply:
Write as `flags |= VIR_MIGRATE_NON_SHARED_DISK`. In general, enabling is flag is more readable as performing an `OR, |` operation than an addition operation.